java/src/main/java/com/google/cloud/dataproc/templates/util/TemplateConstants.java

```diff
@@ -399,4 +399,14 @@ public interface TemplateConstants {
   String SNOWFLAKE_GCS_OUTPUT_FORMAT = "snowflake.gcs.output.format";
   String SNOWFLAKE_GCS_OUTPUT_MODE = "snowflake.gcs.output.mode";
   String SNOWFLAKE_GCS_OUTPUT_PARTITION_COLUMN = "snowflake.gcs.output.partitionColumn";
+
+  /** MongoToGCS properties */
+  String MONGO_GCS_OUTPUT_LOCATION = "mongo.gcs.output.location";
+
+  String MONGO_GCS_OUTPUT_FORMAT = "mongo.gcs.output.format";
+  String MONGO_GCS_OUTPUT_MODE = "mongo.gcs.output.mode";
+  String MONGO_GCS_OUTPUT_MODE_DEFAULT = "append";
```

Comment:
Is this required? I believe it should be handled by templates.properties mongo.gcs.output.mode